utilities/gnmi_cli.cc

```diff
@@ -55,7 +55,7 @@ optional arguments:
 
 #define MAX_STR_LENGTH 128
 
-DEFINE_string(grpc_addr, "0.0.0.0:9339", "gNMI server address");
+DEFINE_string(grpc_addr, "127.0.0.1:9339", "gNMI server address");
```

Comment:
Here also add FIXME comment to indicate that for now it is connecting to localhost and later it has to fix to support any host by providing the options.

/*
  FIXME:  XXX
 */

Reply:
Done, updated comment